Makefile

```diff
@@ -15,10 +15,11 @@
 DOCKER_ARCHS ?= amd64 armv7 arm64 ppc64le s390x
 
 REACT_APP_PATH = web/ui/react-app
-REACT_APP_SOURCE_FILES = $(wildcard $(REACT_APP_PATH)/public/* $(REACT_APP_PATH)/src/* $(REACT_APP_PATH)/tsconfig.json)
+REACT_APP_SOURCE_FILES = $(shell find $(REACT_APP_PATH)/public/ $(REACT_APP_PATH)/src/ $(REACT_APP_PATH)/tsconfig.json)
```

Comment:
Why this change?

Reply:
I got annoyed that changes in the React app frequently didn't get picked up by make because the existing dependency wasn't recursive (we have many subdirs). And `**` is a bit arcane, so I decided to go the `find` way, seems most robust in finding everything.